### DISCUSSION TITLE
Releases page builds for 0.6.0

### DISCUSSION THREAD

**Discussion by tech189** [arxanas/git-branchless#773] [General]
Hi, would it be possible to restore the pre-built binaries in the assets section of the 0.6.0 release? Scoop's package is out of date because of this. Thanks!

**Comment by arxanas** [comment]
If you can figure out a way to trigger the workflow at https://github.com/arxanas/git-branchless/blob/cf94a09ebd826e6c0c1d592940a39bd808d1124e/.github/workflows/release.yml on the latest release, then I'm happy to trigger the workflow for you. I haven't found an easy way to do it myself.
